LiskHQ/lisk-sdk
# Remove previous Round implementation - Closes #3685

### What was the problem?
Rounds module not used anymore

### How did I solve it?
Removed `mem_rounds`, `mem_round_snapshot` and `mem_votes_snapshot` tables.
Removed `modules/chain/round` folder
Removed Storage Round entity
Removed StateStore Round entity
Removed `forks_stat` table and its usages.
Removed/Adjust all tests accordingly

### How to manually test it?
Make sure the application starts.
Make sure it sync with testnet at least up to height 20000.
All tests should be green

### Review checklist

- [ ] The PR resolves #3685
